sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample_AnalyzePrebuiltDocument.md

```diff
@@ -32,7 +32,8 @@ AnalyzeResult result = operation.Value;
 
 foreach (DocumentPage page in result.Pages)
 {
-    Console.WriteLine($"Document Page {page.PageNumber} has {page.Lines.Count} line(s) and {page.Words.Count} word(s).");
+    Console.WriteLine($"Document Page {page.PageNumber} has {page.Lines.Count} line(s), {page.Words.Count} word(s),");
+    Console.WriteLine($"and {page.SelectionMarks.Count} selection mark(s).");
```

Comment:
nit, this will be displayed in 2 lines. In case that was not the intention, I suggest doing:
> Console.Write($"Document Page {page.PageNumber} has {page.Lines.Count} line(s), {page.Words.Count} word(s), ");
    Console.WriteLine($"and {page.SelectionMarks.Count} selection mark(s).");